x/emissions/keeper/inference_synthesis/forecast_implied_inferences_test.go

```diff
@@ -6,67 +6,15 @@ import (
 	emissions "github.com/allora-network/allora-chain/x/emissions/types"
 )
 
-func (s *InferenceSynthesisTestSuite) TestGradient() {
```

Comment:
are we not doing gradient anymore? or we just have to rewrite this test with new numbers because the formula changed

Reply:
I guess we are deleting gradient in forecast_implied_inferences.go

do we still need the params GradientDescentMaxIters and MaxGradientThreshold ? should they be renamed to something else?

Reply:
1) I moved the gradient function to `math/utils`
2) Yeah we are using in `GetAllReputersOutput`